Header header
int16 x1
int16 y1
int16 x2
int16 y2
string obj_class
float32 score
